### ISSUE TITLE
[BUG] AttributeError: 'LazySchema' object has no attribute 'given'

### ISSUE THREAD

**Issue by dkbarn** [schemathesis/schemathesis#1093]
I am trying to write my first test using the examples provided in the schemathesis documention.

I am using ```schemathesis.from_pytest_fixture()``` to create my schema, because my WSGI app itself is defined as a pytest fixture.

Now I'm trying to use ```@schema.given``` to tweak some of the test data using the hypothesis ```data()``` strategy, as shown in the docs here: https://schemathesis.readthedocs.io/en/stable/python.html#using-additional-hypothesis-strategies

When I do this, I get an error: ```AttributeError: 'LazySchema' object has no attribute 'given'```

But I've not encountered anything in the docs saying that this shouldn't work.

**To Reproduce**
```
from hypothesis import strategies
import pytest
import schemathesis

@pytest.fixture()
def wsgi_app_schema(app):
    return schemathesis.from_wsgi("/", app)

schema = schemathesis.from_pytest_fixture("wsgi_app_schema")

@schema.parametrize()
@schema.given(data=strategies.data())
def test(case):
    response = case.call_wsgi()
    case.validate_response(response)
```

**Expected behavior**
```@schema.given``` works regardless of which loader was used to produce the schema.

**Environment:**
 - OS: Linux
 - Python version: 3.7.7
 - Schemathesis version: 3.5.3
 - Spec version: Open API 3.0.2

**Comment by Stranger6667** [comment]
Thank you very much for a great report!

I forgot to implement it for `LazySchema`. Will fix in the next release

**Comment by dkbarn** [comment]
Thanks so much for the super quick fix! I've pulled down master branch and verified that ```@schema.given``` is working for me now.

**Comment by Stranger6667** [comment]
Awesome! I am glad to hear that :)

**Comment by dkbarn** [comment]
Just wanted to check on when you're planning to put out the next release of schemathesis which will include this fix?
Thanks

**Comment by Stranger6667** [comment]
During this week :) Likely tomorrow

**Comment by Stranger6667** [comment]
Released in `3.6.0`

**Comment by dkbarn** [comment]
Thanks! Confirmed that 3.6.0 has resolved the issue.
